packages/pagebuilder/lib/resolveLinkProps.js

```diff
@@ -9,7 +9,7 @@ export default link => {
 
     try {
         const baseUrlObj = new URL(process.env.MAGENTO_BACKEND_URL);
-        const urlObj = new URL(link);
+        const urlObj = new URL(link, baseUrlObj);
```

Comment:
After this bugfix, buttons with hasgtags are not working, and redirect to a homepage.
E.g. in pagebuilder URL: "#test" -> on click you will be redirected to a home page.